cmd/dfget/app/root.go

```diff
@@ -237,6 +237,10 @@ func initFlags() {
 		"be verbose")
 
 	// pass to server
+	flagSet.StringVar(&cfg.RV.LocalIP, "ip", "",
```

Comment:
Which server? Could you be more specific?
The command description is user-facing, and maybe some user who are not so familiar with dragonfly yet will use the command as well. We should treat such kind of user very well, to make them all clear to the parameter's usage.

Reply:
Yeah, I will update the comment to make it more clear.